src/scss/footer/_footer.scss

```diff
@@ -0,0 +1,160 @@
+.footer {
+  background-color: $footerGray;
+
+  &-toggle-list {
+    border-bottom: 1px solid $footerDarkGrey;
+    font-weight: bold;
+    line-height: 30px;
+    padding: 20px 23px;
+  }
+}
+
+.footer-category {
+  border-bottom: 1px solid $footerDarkGrey;
+  font-weight: bold;
+  line-height: 30px;
+  margin: 0px;
+  padding: 20px 23px;
+  &:last-child {
+    border-bottom: 0px;
+  }
+  &__label {
+    margin: 0px;
+  }
+  &__open {
+    fill: $footerCrossColor;
+    float: right;
+    font-size: 30px;
+    height: 36px;
+    vertical-align: middle;
+    width: 36px;
+  }
+  &__open:after {
+    clear: both;
+  }
+}
+
+.js-footer-list {
+  &-is-hiden {
+    display: none;
+  }
+}
+
+.footer-list {
+  padding: 20px 23px 23px 0px;
+  border-bottom: 1px solid $footerDarkGrey;
+  &:last-child {
+    border-bottom: 0px;
+    padding-bottom: 0px;
+  }
+  &:before {
+    content: initial;
+  }
+  &__link {
+    color: $baseFontColor;
+    &:link, &:hover, &:active  {
```

Comment:
same comment as I had. Selector by comma should be in new line https://github.com/airbnb/css#formatting